cool_seq_tool/mappers/mane_transcript.py

```diff
@@ -574,23 +588,48 @@ async def get_longest_compatible_transcript(
     ) -> Optional[
         Union[DataRepresentation, CdnaRepresentation, ProteinAndCdnaRepresentation]
     ]:
-        """Get longest compatible transcript from a gene.
-        Transcript is compatible if it passes validation checks.
+        """Get longest compatible transcript from a gene. See the documentation for
+        the :ref:`transcript compatibility policy <transcript_compatibility>` for more
+        information.
+
+        >>> from cool_seq_tool.app import CoolSeqTool
+        >>> from cool_seq_tool.schemas import AnnotationLayer, ResidueMode
+        >>> mane_mapper = CoolSeqTool().mane_transcript
+        >>> mane_transcripts = {
+        ...     "ENST00000646891.2",
+        ...     "NM_001374258.1",
+        ...     "NM_004333.6",
+        ...     "ENST00000644969.2",
+        ... }
+        >>> result = await mane_mapper.get_longest_compatible_transcript(
+        ...     599,
+        ...     599,
+        ...     gene="BRAF",
+        ...     start_annotation_layer=AnnotationLayer.PROTEIN,
+        ...     residue_mode=ResidueMode.INTER_RESIDUE,
+        ...     mane_transcripts=mane_transcripts,
+        ... )
+        >>> result.refseq
+        'NP_001365396.1'
+
+        If unable to find a match on GRCh38, this method will then attempt to drop down
+        to GRCh37.
+
+        # TODO example for inputs that demonstrate this?
```

Comment:
@korikuzma do you have an example set of inputs handy for this?

Reply:
Not off the top of my head. We can make a separate issue if needed.

Reply:
(sorry, this duplicates a comment you had above. Probably not necessary if there isn't a good example available -- I was just trying to illustrate different code branches based on input)

Reply:
No worries. It is good to have an example and see if our test suite covers it. I just don't know off the top of my head and would have to dig into the test data